airbyte-integrations/connectors/source-tiktok-marketing/source_tiktok_marketing/manifest.yaml

```diff
@@ -22,9 +22,11 @@ definitions:
     error_handler:
       type: DefaultErrorHandler
       response_filters:
-        - predicate: "{{ response.get('code') != 0 }}"
+        - predicate: "{{ response.get('code') != 0 and response.get('code') not in (40100, 50002) }}"
           action: FAIL
           error_message: "{{ response['message'] }}"
+        - predicate: "{{ response.get('code') == 40100 }}"
+          action: RATE_LIMITED
```

Comment:
Can we avoid duplicating of `40100` code in both predicates by moving 
```
- predicate: "{{ response.get('code') == 40100 }}"
  action: RATE_LIMITED
```
to the first place in response_filters list? 
It looks like DefaultErrorHandler should return first match for predicates, so it's no need to add `40100` code for FAIL action.